lib/src/fade.dart

```diff
@@ -7,6 +7,8 @@ import 'animated_text.dart';
 class FadeAnimatedText extends AnimatedText {
   final double fadeInEnd, fadeOutBegin;
 
+  /// fadeInEnd marks ending of fade-in interval, default value = 0.5
+  /// fadeOutBegin marks starting of fade-out interval, default value = 0.8
```

Comment:
The comments need to be moved above the declaration.  (The DartDoc needs to document the declarations.)
Should probably split them too, like:
```dart
  /// Marks ending of fade-in interval, default value = 0.5
  final double fadeInEnd;
  /// Marks the beginning of fade-out interval, default value = 0.8
  final double fadeOutBegin;
```

Reply:
Oh ok. Done :+1: